source/shared/cpp/ObjectModel/Container.cpp

```diff
@@ -56,6 +57,28 @@ void Container::SetSelectAction(const std::shared_ptr<BaseActionElement> action)
     m_selectAction = action;
 }
 
+void Container::SetLanguage(const std::string& value)
+{
+    for (auto& item : m_items)
+    {
+        CardElementType elementType = item->GetElementType();
+
+        if (elementType == CardElementType::Container)
+        {
+            auto element = std::static_pointer_cast<Container>(item);
+            element->SetLanguage(value);
+        }
+        else
+        {
+            if (elementType == CardElementType::TextBlock)
```

Comment:
just make this:
else if (elementType == CardElementType::TextBlock)

Reply:
aside from containers, show card actions and column sets, are there some other elements that can have a Text block inside of it?

---
In reply to: [170754476](https://github.com/Microsoft/AdaptiveCards/pull/1102#discussion_r170754476) [](ancestors = 170754476)